docs/kafka/quarkus-kafka/README.adoc

```diff
@@ -228,9 +229,9 @@ The Quarkus example application in this quick start has three application-scoped
 $ cd ~/code-examples/quarkus-kafka-quickstart
 $ ./mvnw quarkus:dev
 ----
-. After the application is running, in a web browser, go to http://localhost:8080/prices.html[^] and verify that the `Last price` is updated.
+. When the application is running, in a web browser, go to http://localhost:8080/prices.html[^] and verify that the `Last price` value is updated.
```

Comment:
This sentence is a little awkward to read. Suggestion for increased readability:
When the application is running, perform the following actions:
-  Open a web browser and go to http://localhost:8080/prices.html. 
-  Verify that the `Last price` value is updated.

Reply:
Yes, that's a nice update.